pkg/communication/dialer.go

```diff
@@ -1,7 +1,7 @@
 // Copyright (C) 2019 Storj Labs, Inc.
 // See LICENSE for copying information.
 
-package kademliaclient
+package communication
```

Comment:
Note, for the actual PR, separate the deletion from the changes and renames. It's easier to approve deletion.

Reply:
Ok sounds good